narayana-release-process.sh

```diff
@@ -42,7 +40,7 @@ then
     exit
   fi
 fi
-read -p "You will need: VPN, credentials for jbosstm@${RSYNC_HOST}, jira admin, github permissions on all jbosstm/ repo and nexus permissions. Do you have these?" ENVOK
+read -p "You will need: VPN, credentials for jbosstm host, jira admin, github permissions on all jbosstm/ repo and nexus permissions. Do you have these?" ENVOK
```

Comment:
Just guessing, but I think probably ", credentials for jbosstm host" can be removed?

Reply:
Actually I think that is still a requirement to update the website. In fact when the PR is approved (e.g. https://github.com/jbosstm/narayana.io/pull/190)  it is still needed to update the website with the new website change (rsync), which needs an authorized account. We could also create a github action to do that automatically if we want to.

Reply:
Thanks for the insight, we can resolve my guess-work